pytest_selenium/pytest_selenium.py

```diff
@@ -243,7 +244,7 @@ def _gather_logs(item, report, driver, summary, extra):
     pytest_html = item.config.pluginmanager.getplugin('html')
     if item.config._driver_log and os.path.exists(item.config._driver_log):
         if pytest_html is not None:
-            with open(item.config._driver_log, 'r') as f:
+            with io.open(item.config._driver_log, 'r', encoding = 'utf8') as f:
```

Comment:
The flake8 build is failing because of the spaces around the ```=``` after 'encoding'. 
It should be ```encoding='utf-8'```